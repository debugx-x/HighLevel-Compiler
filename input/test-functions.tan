subr int fin( int a, int b ){
	return pax(5*a, b);
}
subr int pax( int a, int b ){
	if( a < b ){
		return a;
	}
	return b + pax(a, 3*b);	
}
main {
	const a := 11;
	const b := 14;
	const c := 16;
	
	if( a > 5){
		const m := pax(a, b);
		print m \n;
		const pax := fin(a, m);
		print pax\n;
	}
	const x := pax(a,b);
	print x \n;
}