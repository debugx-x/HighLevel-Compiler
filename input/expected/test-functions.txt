Not enough operands on stack for StoreI instruction at location 116.
